packages/components/spacings/spacings-inline/src/inline.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { filterDataAttributes } from '@commercetools-uikit/utils';
+import getStyles from './inline.styles';
+import type { TProps } from './types';
```

Comment:
I prefer to have the `import type` imports before the normal imports

Reply:
The thing about this is that this is also needed in `inline.styles` file.
Should I then import it there from the this component file?